datasets/lv/sanction_list/lv_sanction_list.yml

```diff
@@ -0,0 +1,28 @@
+name: lv_sanction_list
+title: Latvia List of Sanctions
+entry_point: crawler.py
+prefix: lv-sanction
+summary: List of Sanctions in Latvia
+description:
+  This dataset contains lists of individuals or entities subject to international or national sanctions as specified by laws in Latvia
+
+  International sanctions are imposed on subject in accordance to international laws as set by organizations such as the UN, EU or other international bodies.
+
```

Comment:
we're only doing the national sanctions, right? So I think let's change this to reflect that - no need even to mention that they also publish UN and EU sanctions